### DISCUSSION TITLE
A bit confuse about add_commands() and other config in readme

### DISCUSSION THREAD

**Discussion by VisionaryAppDev** [mfussenegger/nvim-jdtls#330] [Q&A]
Hello everyone,

I am a bit confuse with the below config from README.md so could any one explain what is the purpose of the below config? And is it different from `require('jdtls.setup').add_commands()`.

```
command! -buffer -nargs=? -complete=custom,v:lua.require'jdtls'._complete_compile JdtCompile lua require('jdtls').compile(<f-args>)
command! -buffer -nargs=? -complete=custom,v:lua.require'jdtls'._complete_set_runtime JdtSetRuntime lua require('jdtls').set_runtime(<f-args>)
command! -buffer JdtUpdateConfig lua require('jdtls').update_project_config()
command! -buffer JdtJol lua require('jdtls').jol()
command! -buffer JdtBytecode lua require('jdtls').javap()
command! -buffer JdtJshell lua require('jdtls').jshell()
```

Thanks

**Comment by mfussenegger** [comment]
It's just listing what `add_commands()` does for you.

> Some methods are better exposed via commands. As a shortcut you can also call :lua require('jdtls.setup').add_commands() to declare these.

**Reply by VisionaryAppDev** [reply]
I see. So, they are the same but just we could use `add_commands()` as a short form.
